spec/models/project_spec.rb

```diff
@@ -21,6 +21,19 @@
     expect(sub_project2.position).to eq 2
   end
 
+  describe ".with_ordered_stories" do
+    it "orders sub projects properly" do
+      parent = FactoryBot.create(:project)
+      sub_project1 = FactoryBot.create(:project, parent: parent)
+      FactoryBot.create(:story, project: sub_project1, position: 2)
+      sub_project2 = FactoryBot.create(:project, parent: parent)
+      FactoryBot.create(:story, project: sub_project2, position: 1)
```

Comment:
I think the test is not correct, the problem we have is that we can have multiple stories between all the subprojects with the same position (like story 1 in subproject 1 and story 1 in subproject 2), it's working in this case because we are setting 2 different positions but in 2 real projects we would have a lot or repeated positions (though unique in the scope of each subproject)

This should order subprojects by the sub-project's position field, not the stories one